cloudsql/mysql/database-sql/deployment.yaml

```diff
@@ -0,0 +1,66 @@
+# [START cloud_sql_mysql_databasesql_gke_quickstart_deployment]
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gke-cloud-sql-quickstart
+spec:
+  selector:
+    matchLabels:
+      app: gke-cloud-sql-app
+  template:
+    metadata:
+      labels:
+        app: gke-cloud-sql-app
+    spec:
+      serviceAccountName: <YOUR-KSA-NAME>
+      containers:
+      - name: gke-cloud-sql-app
+        # Replace $LOCATION with your Artifact Registry location (e.g., us-central1).
+        # Replace $YOUR_PROJECT_ID with your project ID.
+        image: $LOCATION-docker.pkg.dev/$YOUR_PROJECT_ID/gke-cloud-sql-repo/gke-cloud-sql-image:latest

```

Comment:
Why the difference in variable declarations, $LOCATION vs <LOCATION>?

Reply:
ditto below

Reply:
Updated all variable placeholders to use same format.